Unity-Technologies/ScriptableRenderPipeline
# Testing XR PostFX working on URP

### Purpose of this PR
PostFX are broken on current 7.1.x and master. This PR adds tests that show that and prevent it from happening again.

### Release Notes
Adds four different XR tests.
- toggling "Use Postprocessing" on on a camera
- using FXAA
- using SMAA
- adding a Volume to the scene with Bloom / Depth of Field that don't actually do something

### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

Reference image for all tests:
![013_xr_PostFX_Simple](https://user-images.githubusercontent.com/2693840/65257279-44f70380-db01-11e9-9223-38bcd8cb8c08.png)
_Note: the tests for FXAA and SMAA should look different but since they totally fail currently I included this as baseline result. These reference images should be updated once the basics work and have a really small tolerance to detect failing FXAA/MSAA._

Actual current results:

**013**
_Note that 013 currently succeeds despite the actual & result images looking very different. This is a bug in the Graphics Test Framework. Workaround is reducing the tolerance on the camera. Additionally, the result during running the test differs from the result seen when pressing Play._
![013_xr_PostFX_Simple](https://user-images.githubusercontent.com/2693840/65257559-b0d96c00-db01-11e9-8570-50223a5a3079.png)

**014**
![014_xr_PostFX_Complex_NoAA](https://user-images.githubusercontent.com/2693840/65257561-b0d96c00-db01-11e9-8df5-9de822a8ff02.png)

**015**
![015_xr_PostFX_FXAA](https://user-images.githubusercontent.com/2693840/65257565-b0d96c00-db01-11e9-843a-feedb285da50.png)

**016**
_Note that 016 doesn't fail because of image content but because of the depth buffer not being the same size as the color buffer._
![016_xr_PostFX_SMAA](https://user-images.githubusercontent.com/2693840/65257566-b1720280-db01-11e9-95ab-605cc966874a.png)

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
None

**Halo Effect**: None, Low, Medium, High?
Low, increases test run time

### Comments to reviewers
Note that the numbering starts with 013 because of my previous pull request, #4675.
Also please note that you should run these tests with all of the available Stereo modes (Multi Pass, Single Pass, Single Pass Instanced) since they all fail in different ways.
